rlclientlib/schema/v2/Metadata.fbs

```diff
@@ -1,6 +1,6 @@
 namespace reinforcement_learning.messages.flatbuff.v2;
 
-enum PayloadType : ubyte { CB, CCB, Slates, Outcome, CA, DedupInfo, MultiStep }
+enum PayloadType : ubyte { CB, CCB, Slates, Outcome, CA, DedupInfo, MultiStep, Episode }
```

Comment:
@ataymano Perhaps need to organize the naming a bit:
- Episode -> episode id
- Multistep -> interaction event in an episode
- Outcome -> observation event in an episode